core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -885,17 +907,7 @@ protected void selectTab(int position) {
     }
   }
 
-  private KiwixWebView safelyGetWebView(int position) {
```

Comment:
these lines shouldn't be deleted either, I think you had a bad merge with master maybe before it picked up this file as a rename instead of a new file. thankfully it did recognise it as a rename so we can see these minor flaws